Automattic/wp-calypso
# Add/link in bio setup

#### Proposed Changes

This PR adds a basic implementation of the second (setup) step on the link in bio flow

**What is included on this PR?**

- It should look like the [design](https://www.figma.com/file/BcUdhLlhAp7UHtQHNTI4zA/Tailored-Onboarding-%26-Launchpad?node-id=1890%3A22166)
- Error messages are hidden but there if `isValid` is equal `false`.

**How to test**
- Run `yarn start` from the calypso root folder
- Navigate to `http://calypso.localhost:3000/setup/linkInBioSetup?flow=link-in-bio`
- Or to see the flow from the start, navigate `http://calypso.localhost:3000/setup/intro?flow=link-in-bio` then click o "Get Started"

**What is not implemented**

- Form validation
- Redux store
- Icon upload ( soon will be integrated/reused with the same component being used on the newsletter flow )


**Visual reference:**

<img width="406" alt="image" src="https://user-images.githubusercontent.com/2653810/182365917-ad932bc6-6337-485b-b10f-e2952d72dd09.png">

<img width="978" alt="image" src="https://user-images.githubusercontent.com/2653810/182365992-1564e774-839c-4aa9-ab96-3c8d60d601a6.png">


<img width="574" alt="image" src="https://user-images.githubusercontent.com/2653810/182404239-30f4a387-5c89-4c5b-a4d7-a1eebebfc74e.png">


Related to #66163
Fixes #66163
